app/api/srch/search.rb

```diff
@@ -107,8 +107,9 @@ def self.execute(endpoint, params)
       tag_query = params[:tagName]
       order_query = params[:order_direction]
       sort_query = params[:sort_by]
+      field_query = params[:field]
       search_type = endpoint
-      search_criteria = SearchCriteria.new(search_query, tag: tag_query, sort_by: sort_query, order_direction: order_query)
+      search_criteria = SearchCriteria.new(search_query, tag: tag_query, sort_by: sort_query, order_direction: order_query, field: field_query)
```

Comment:
Do you think we could break this up over multiple lines? It could be indented so they line up. And is there an advantage to making variables to store these values in the above lines versus directly using params[] in the .new call?